pgh_play/www/php/interface.php

```diff
@@ -201,5 +203,72 @@ function get_parks(){
 	return $results;
 }
 
+// enter new park into the db via new park form
+// this will store the following info in the db: Park name, address, rating, ages, comments
+//  all of these inputs are received by the new park form a user fills out
+// *****facilities and pictures still need to be implemented 
+function new_park(){
```

Comment:
I created the function to coincide with the simple new park form I whipped together. I agree that it probably needs refactored, but I wanted to get the functionality working with the db before cleaning it up and drilling down more into separate functions